tox.ini

```diff
@@ -40,9 +39,6 @@ commands =
 
 [testenv:build]
 
-[testenv:codecov]
-commands = - codecov -e TOX_VENV,ZOOKEEPER_VERSION
```

Comment:
It looks like previously we ran coverage separately for each matrix entry... so removing this we'll only run it once, which means we lose visibility if a particular python/ zookeper version combo loses coverage unexpectedly due to a special casing in the code...

I personally think that's fine, coverage is more a "nice-to-have" tool to surface things, and rather have it run quickly than exhaustively, but wanted to document this.